templatestore/frontend/src/components/PreviewButton.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable
```

Comment:
Can we remove this?

Reply:
Yes, this file is removed in the newer commits.